metricbeat/module/elasticsearch/node/data.go

```diff
@@ -93,8 +93,7 @@ func eventsMapping(r mb.ReporterV2, info elasticsearch.Info, content []byte) err
 			continue
 		}
 
-		// Write name here as full name only available as key
-		event.MetricSetFields["name"] = name
```

Comment:
We remove the `name` field but it still appears in `data.json`, is it  being set also in other place?

Reply:
Correct, it is being set here: https://github.com/elastic/beats/pull/9261/files#diff-a32493166e94b045c0eba2705d5744b1R35.

We were previously incorrectly setting it to the ID. See the related conversation here: https://github.com/elastic/beats/pull/9209#discussion_r235816937